### PR TITLE
refactor: move remote WAL under the disaster recovery

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

as title

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Comment by sunng87** [comment]
remote wal is not for DR only, we'd better keep it standalone.

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
     "description": "The label for category Disaster Recovery in sidebar docs"
   },
   "sidebar.docs.category.Remote WAL": {
-    "message": "Remote WAL",
+    "message": "远程 WAL",
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:67]: `Remote` is fine; translation isn't needed.

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/admin.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ ORDER BY datanode_id ASC
 ## 数据管理
 
 * [存储位置说明](/user-guide/concepts/storage-location.md)。
-* 通过 [设置Remote WAL](./remote-wal/quick-start.md) 实现 GreptimeDB 的集群容灾。
+* 通过 [设置Remote WAL](/user-guide/operations/disaster-recovery/remote-wal/quick-start.md) 实现 GreptimeDB 的集群容灾。
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/admin.md:39]:
```suggestion
* 通过 [设置 Remote WAL](/user-guide/operations/disaster-recovery/remote-wal/quick-start.md) 实现 GreptimeDB 的集群容灾。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/overview.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ GreptimeDB 基于存储计算分离的云原生架构设计：
 
 * **Frontend**：数据插入和查询的服务层，将请求转发到 Datanode 并处理和合并 Datanode 的响应。
 * **Datanode**：GreptimeDB 的存储层，是一个 LSM 存储引擎。Region 是在 Datanode 中存储和调度数据的基本单元。Region 是一个表分区，是一组数据行的集合。Region 中的数据保存在对象存储中（例如 AWS S3）。未刷新的 Memtable 数据被写入 WAL，并可以在灾难发生时恢复。
-* **WAL**：持久化内存中未刷新的 Memtable 数据。当 Memtable 被刷新到 SSTable 文件时，WAL 将被截断。它可以是基于本地磁盘的（本地 WAL）或基于 Kafka 集群的（远程 WAL）。
+* **WAL**：持久化内存中未刷新的 Memtable 数据。当 Memtable 被刷新到 SSTable 文件时，WAL 将被截断。它可以是基于本地磁盘的（本地 WAL）或[基于 Kafka 集群的（远程 WAL）](./remote-wal/quick-start.md)。
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/overview.md:45]:
```suggestion
* **WAL**：持久化内存中未刷新的 Memtable 数据。当 Memtable 被刷新到 SSTable 文件时，WAL 将被截断。它可以是基于本地磁盘的（Local WAL）或[基于 Kafka 集群的（Remote WAL）](./remote-wal/quick-start.md)。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/remote-wal/quick-start.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 # 快速开始
 
-## 什么是 Remote WAL
+## 什么是远程 WAL（Remote WAL）
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/remote-wal/quick-start.md:3]:
```suggestion
## 什么是 Remote WAL
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/remote-wal/quick-start.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
 
 [WAL](/contributor-guide/datanode/wal.md#introduction)(Write-Ahead Logging) 是 GreptimeDB 中的一个关键组件，它持久记录每一次数据修改，以确保不会丢失缓存在内存中的数据。我们在  [Datanode](/user-guide/concepts/why-greptimedb.md)  服务中用持久的嵌入式存储引擎 [raft-engine](https://github.com/tikv/raft-engine) 将 WAL 实现为一个模块。在公共云中部署 GreptimeDB 时，我们可以在云存储（AWS EBS、GCP 持久盘等）中持久存储 WAL 数据，以实现 0 RPO。然而，由于 WAL 与 Datanode 紧密耦合，导致部署过程中的 RTO（Recovery Time Objective）较长。此外，由于 raft-engine 无法支持多日志订阅，这使得实现 region 热备份和 region 迁移变得困难。
 
 为了解决上述问题，我们决定设计并实现一个远程 WAL。远程 WAL 将 WAL 从 Datanode 分离到远程服务，我们选择了 Apache Kafka 作为远程服务。Apache Kafka 在流处理中被广泛采用，展现出卓越的分布式容错能力和基于主题的订阅机制。在发布 v0.5.0 版本时，我们引入了 Apache Kafka 作为 WAL 的可选存储引擎。
 
 
-## 运行带有 Remote WAL 的 Standalone GreptimeDB
+## 运行带有远程 WAL 的 Standalone GreptimeDB
```
Comment (fengjiachun) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/operations/disaster-recovery/remote-wal/quick-start.md:10]:
```suggestion
## 运行带有 Remote WAL 的 Standalone GreptimeDB
```

**Comment by nicecui** [comment]
> remote wal is not for DR only, we'd better keep it standalone.

**Pull request closed without merging**